api/pfcpsim.proto

```diff
@@ -13,9 +13,9 @@ message CreateSessionRequest {
   int32 baseID = 2;
   string nodeBAddress = 3;
   string ueAddressPool = 4;
-  string sdfFilter = 5;
+  string appFilter = 5;
   int32 qfi = 6; // Should be uint8
-  bool gateClosed = 7;
+  bool gateClosed = 7; // If set, the application QERs will have the Gate status to closed.
```

Comment:
Isn't `gateClosed` in conflict with `appFilter` now? What if we define gateClosed = true and appFilter = allow ? Which one have a precedence?

Reply:
Right. I removed the ability to set the gate status. The only way to set the gate status is now by using the app-filter flag